src/Prefixer.js

```diff
@@ -8,7 +8,7 @@ import {hash, charat, strlen, indexof, replace, substr, match} from './Utility.j
  * @return {string}
  */
 export function prefix (value, length, children) {
-	switch (hash(value, length)) {
+	switch (charat(value, 0) != 45 && charat(value, 1) != 45 && hash(value, length)) {
```

Comment:
the expression's type now is `false | number` - perhaps we should convert `false` to a `0`? would that have any effect on the engine? According to some TS PRs that I've seen lately (like [this one](https://github.com/microsoft/TypeScript/pull/50225)) JS engines don't do jump-table optimization for switch statements anyway so perhaps this doesn't matter at all? I'm not sure 🤷‍♂️

Reply:
The second `charat(value, 0) != 45` is probably not needed as we also prefer to not touch `-webkit-` already prefixed props. 

It'd put `charat(value, 0) ^ 45 ? 0 : ...` inside the `hash` function.

Reply:
Done